README.md

```diff
@@ -131,6 +132,15 @@ final report.
 Flaky tests are also detected and printed as such in the final report. They are
 also emitted to Sentry (see [Sentry integration](#sentry-integration)).
 
+### Fail-Fast
+
+In order to prevent large suites running, a threshold can be set to control the
```

Comment:
`running for a long time with a log of failures,` I think it requires to describe the conditions under which someone whould/should use this parameter.

Reply:
3b47fe8